lib/src/test/kotlin/at/bitfire/ical4android/validation/FixInvalidDayOffsetPreprocessorTest.kt

```diff
@@ -12,8 +12,7 @@ import java.time.Duration
 
 class FixInvalidDayOffsetPreprocessorTest {
 
-    private fun fixAndAssert(expected: String, testValue: String) {
-
+    private fun fixParseAssertEqual(expected: String, testValue: String) {
```

Comment:
This method can be used for all tests, true? At least when I replace every `fixAssertNotEqual` call by `fixParseAssertEqual`, all tests still succeed.

Can we have only one assert method for all cases? Otherwise we should find a better naming, and maybe also KDoc for the assert helpers so that it's clear what they do.

See also next comment.